README.md

```diff
@@ -1 +1,460 @@
-# golangcodestyle
\ No newline at end of file
+# Backend (Golang) Coding Guidelines
```

Comment:
3. Try to use `t.Parallel()` for the top level tests and the table sub-tests when possible.

Reply:
added: https://github.com/apaliavy/golangcodestyle/blob/1eda2a9506765ca11274f4bf973baf16a84174af/README.md#parallelism